Ontime.js

```diff
@@ -88,13 +112,13 @@ function wsMessageReceived(message) {
   } else if (type == 'ontime-runtime') {
     var runtime = local.values.runtime;
 
-    runtime.selectedEventIndex.set(payload.selectedEventIndex);
+    runtime.activeEventIndex.set((payload.selectedEventIndex >= 0) ? payload.selectedEventIndex + 1 : -1); // Off by 1, starts at 0, -1 when inactive
```

Comment:
I'm unsure if 1 indexing is a good idea?
What is normally done in other Chataigne nodules?

Reply:
The idea is mainly to match the index displayed in Ontime.

Triggers sent to Ontime follows that index so it might feel a bit unclear that you would recieve index 0 from Ontime when the first event runs but you send trigger 1 to start the same event..

However, maybe it can/should then be 0 instead of -1 when no event is active..

Reply:
> Triggers sent to Ontime follows that index so it might feel a bit unclear that you would recieve index 0 from Ontime when the first event runs but you send trigger 1 to start the same event..

Ahh yeah you are right about that

I also see that the index is not cleared properly when nothing is selected 
this will need to be fixed upstream